components/Header.jsx

```diff
@@ -1,11 +1,20 @@
 import AccountInfo from "@/components/AccountInfo";
 import styles from "@/styles/components/header.module.scss";
+import { useRouter } from "next/router";
 
 export default function Header() {
+	const router = useRouter();
+	const style = {
+		display: router.asPath === "/" ? "none" : "block",
```

Comment:
1. `style` is not a descriptive name. It would work if it were for the parent container - as it would represent the whole component. Moreover it's confusing because you call your module `styles`, and the difference is very small and does not properly explain their differences.
2. The first point was for general/future uses, but in this case you shouldn't hide your a element using styles. You want to remove the element from the dom tree if it's not used:
- It's conceptually wrong - you shouldn't have an element rendered if it's not going to be used.
- Accessibility wise, screen readers will still read your element as it's still there.

You need to conditionally render the element.

Reply:
noted

Reply:
note that I mentioned that you need to conditionaly render the _element_ not the component. If you do things this way, you run into the same problem as before as you duplicate your exact same code for the most part.

conditionly render only the element.

Reply:
noted